runelite-client/src/main/java/net/runelite/client/plugins/timetracking/OverviewTabPanel.java

```diff
@@ -68,21 +69,17 @@
 		birdHouseOverview = new OverviewItemPanel(itemManager, pluginPanel, Tab.BIRD_HOUSE, "Bird Houses");
 		add(birdHouseOverview);
 
-		farmingOverviews = new LinkedHashMap<>();
-		farmingOverviews.put(PatchImplementation.HERB, new OverviewItemPanel(itemManager, pluginPanel, Tab.HERB, "Herb Patches"));
-		farmingOverviews.put(PatchImplementation.TREE, new OverviewItemPanel(itemManager, pluginPanel, Tab.TREE, "Tree Patches"));
-		farmingOverviews.put(PatchImplementation.FRUIT_TREE, new OverviewItemPanel(itemManager, pluginPanel, Tab.FRUIT_TREE, "Fruit Tree Patches"));
-		farmingOverviews.put(PatchImplementation.SEAWEED, new OverviewItemPanel(itemManager, pluginPanel, Tab.SPECIAL, "Special Patches"));
-		farmingOverviews.put(PatchImplementation.FLOWER, new OverviewItemPanel(itemManager, pluginPanel, Tab.FLOWER, "Flower Patches"));
-		farmingOverviews.put(PatchImplementation.ALLOTMENT, new OverviewItemPanel(itemManager, pluginPanel, Tab.ALLOTMENT, "Allotment Patches"));
-		farmingOverviews.put(PatchImplementation.BUSH, new OverviewItemPanel(itemManager, pluginPanel, Tab.BUSH, "Bush Patches"));
-		farmingOverviews.put(PatchImplementation.GRAPES, new OverviewItemPanel(itemManager, pluginPanel, Tab.GRAPE, "Grape Patches"));
-		farmingOverviews.put(PatchImplementation.HOPS, new OverviewItemPanel(itemManager, pluginPanel, Tab.HOPS, "Hops Patches"));
-
-		for (OverviewItemPanel panel : farmingOverviews.values())
-		{
-			add(panel);
-		}
+		farmingOverviews = Stream.of(Tab.FARMING_TABS)
+			.filter(v -> v != Tab.OVERVIEW)
+			.collect(ImmutableMap.toImmutableMap(
+				Function.identity(),
+				t ->
+				{
+					OverviewItemPanel p = new OverviewItemPanel(itemManager, pluginPanel, t, t.getName() + " Patches");
+					add(p);
```

Comment:
Is add necessary? Based on documentation, simply returning the value is enough

Reply:
`add` is for the JPanel. Without it nothing farming is in the overview tab.